plugin/storage/badger/spanstore/writer.go

```diff
@@ -193,7 +190,7 @@ func createTraceKV(span *model.Span, encodingType byte, startTime uint64) ([]byt
 	return key, bb, err
 }
 
-// Close Implements io.Closer and closes the underlying storage
+// Close Implements io.Closer but does nothing.
 func (w *SpanWriter) Close() error {
-	return w.closer.Close()
+	return nil
 }
```

Comment:
I think we could also remove this function altogether (will need a minor clean-up of the tests which expect the writer to implement io.Closer).

Reply:
Agree should be removed.

Reply:
Is it only this span writer that doesn't need a closer?

Reply:
Memory storage probably doesn't need a closer either. Even in Cassandra writer the state is in the session that is shared with the reader and thus closed by the factory.

Reply:
Alright, sounds good to me then.